lib/src/facts/linux/os_linux.cc

```diff
@@ -46,6 +46,10 @@ namespace facter { namespace facts { namespace linux {
     {
         // Check for Debian variants
         bs::error_code ec;
+        if (is_regular_file(release_file::huawei, ec)) {
```

Comment:
@MikaelSmith it's Debian 8 PowerPC but they are using sysv instead of systemd. There is an /etc/os-release file which has the details of Debian 8 in it.